linker/shared/src/main/scala/org/scalajs/linker/backend/BasicLinkerBackend.scala

```diff
@@ -97,8 +107,132 @@ final class BasicLinkerBackend(config: LinkerBackendImpl.Config)
       }
     }
 
+    printedModuleSetCache.startRun()
+
     logger.timeFuture("BasicBackend: Write result") {
       writer.write(moduleSet)
+    }.andThen { case result =>
```

Comment:
`_ =>` ?

Reply:
`_ =>` is not enough for it to be considered a `PartialFunction`, which `andThen` demands, but I made it `case _ =>`.